src/main/java/com/thirdcc/webapp/web/rest/DebtResource.java

```diff
@@ -55,7 +55,11 @@ public DebtResource(DebtService debtService) {
     @PreAuthorize("hasRole(\"" + AuthoritiesConstants.ADMIN + "\") || @managementTeamSecurityExpression.isCurrentAdministrator()")
     public ResponseEntity<List<DebtDTO>> getAllDebts(Pageable pageable, @RequestParam MultiValueMap<String, String> queryParams, UriComponentsBuilder uriBuilder) {
         log.debug("REST request to get a page of Debts");
+        System.out.println(pageable);
         Page<DebtDTO> page = debtService.findAllOpenDebts(pageable);
+        for(DebtDTO debt : page){
+            System.out.println(debt);
+        }
```

Comment:
Please remove sout
if really want to have log use log

Reply:
Resolve by https://github.com/Five-Fishes/Club-Management-Spring-Server/pull/59/commits/eac70653eb322e222102c43c5cfaf0b367c757e0